gensrc/thrift/FrontendService.thrift

```diff
@@ -483,6 +483,7 @@ struct TLoadTxnBeginRequest {
     10: optional i64 timeout
     11: optional Types.TUniqueId request_id
     12: optional string auth_code_uuid
+    13: optional string token
```

Comment:
I think we can unify `auth_code_uuid` and `token`.

Currently, `auth_code_uuid` is used for transactional insert operation, and is generated for each TransactionState on FE side, and pass it to BE.
I think we can replace the `auth_code_uuid` with `token`. When creating `TransactionState`, get a token from TokenManager, and when checking token, just check if token is still valid.

Reply:
@morningman In this pr, just keep token filed? will replace `auth_code_uuid ` into `token` cause compatibility issue

Reply:
I think we don't need to handle this compatibility, just keep only one field is ok